StripeUICore/StripeUICore/Source/Elements/Section/SectionElement+MultiElementRow.swift

```diff
@@ -19,7 +19,7 @@ public extension SectionElement {
         public let elements: [Element]
         public let theme: ElementsAppearance
 
-        public init(_ elements: [Element], theme: ElementsAppearance = .default) {
+        public init(_ elements: [Element], theme: ElementsAppearance) {
```

Comment:
Rest of the changes are due to this - it's ~always a bug in production to create this (or any Element) without supplying a theme.